pinot-spi/src/main/java/org/apache/pinot/spi/services/ServiceRole.java

```diff
@@ -22,5 +22,5 @@
  * ServiceRole defines a role that Pinot Service could start/stop.
  */
 public enum ServiceRole {
-  CONTROLLER, BROKER, SERVER, MINION,
+  CONTROLLER, BROKER, SERVER, MINION, ADD_TABLE
```

Comment:
`ADD_TABLE` is not a service in pinot, it should be an external service.

Reply:
ok